.eslintrc

```diff
@@ -8,6 +8,7 @@
   },
   "env": {
     "node": true,
+    "es6": true
```

Comment:
It's actually already enabled for es6. The original file has this in there.
```
{
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module"
  },
``` 
That addition was only about making Promises global.